jobs/ci-run/build/buildjuju.yml

```diff
@@ -12,6 +12,12 @@
       - timestamps
       - build-name:
           name: build-juju-${BUILD_LABEL}
+    properties:
+      - build-discarder:
```

Comment:
Do we need `logrotate` in here?, it's the type for build-discarder. I'm unclear of the differences in how o7k did this and what's here

Reply:
the docs that Harry linked to https://jenkins-job-builder.readthedocs.io/en/latest/properties.html#properties.build-discarder
do seem to say that the key should be build-discarder.

Also, my guess is that he actually ran this, because I did see that 'build-juju' specifically didn't have 90 days 200 runs, but 30 days 50 runs. So if it is doing the right thing, I'm happy.

Reply:
logrotate property is deprecated.